Runtime/Scripts/Skillprint.cs

```diff
@@ -118,5 +118,13 @@ public void LevelRestart(int level)
            new Dictionary<string, dynamic> { ["level"] = level }
         );
     }
+
+    public void Hint(int level)
```

Comment:
Is this sent when we show hint? If yes, should we name it `ShowHint?`

Reply:
IMO letting the method name be 1:1 with the event name is more important ergonomics than forcing the method names to be verbs